packages/global/test/__snapshots__/index.tsx.snap

```diff
@@ -45,32 +45,25 @@ exports[`renders global styles 1`] = `
 <html>
   <head>
     <style
-      data-emotion="css"
-      data-s=""
-    >
-      
-      .css-ck0s41{}
-    </style>
-    <style
-      data-emotion="css"
+      data-emotion="css-global"
       data-s=""
     >
       
       @font-face{font-family:some-name;}
     </style>
     <style
-      data-emotion="css"
+      data-emotion="css-global"
       data-s=""
     >
       
-      .css-ck0s41 body{font-family:Georgia,serif;margin:0;}
```

Comment:
I've read Emotion Global's code to figure it out:

1. This class name shouldn't be there.
2. `data-emotion` should be `css-global`, not `css`.
3. Wait up, original Global uses `styles` instead of `css` prop. Eureka moment.
4. We have JSX pragma configured in our tests.
5. `sx` becomes `css`, which becomes `className`